### PR TITLE
Versioning

### LINKED ISSUE #1153: Task `if` seems to be broken
Experiment.yaml from here: https://github.com/huang195/iter8-gitops/blob/main/experiment.tpl, results in the follow error message.

```
time=2022-01-28 19:10:51 level=info msg=found run task: echo "Promote"
curl -v -H 'Content-Type: application/json' \
    -d '{ "action": "promote", "pull_request": { "merged": "", "commits": "", "base": { "ref": "" }}}' \
    http://el-iter8-eventlistener-default.openshift-f0f5a1e5d9c5f09a7767f1f253010cba-0000.us-east.containers.appdomain.cloud

time=2022-01-28 19:10:51 level=info msg=found run task: echo "Call cleanup"
curl -v -H 'Content-Type: application/json' \
    -d '{ "action": "cleanup", "pull_request": { "merged": "", "commits": "", "base": { "ref": "" }}}' \
    http://el-iter8-eventlistener-default.openshift-f0f5a1e5d9c5f09a7767f1f253010cba-0000.us-east.containers.appdomain.cloud

time=2022-01-28 19:10:51 level=info msg=task 1: gen-load-and-collect-metrics-http : started
time=2022-01-28 19:11:03 level=info msg=task 1: gen-load-and-collect-metrics-http : completed
time=2022-01-28 19:11:03 level=info msg=task 2: assess-app-versions : started
time=2022-01-28 19:11:03 level=info msg=task 2: assess-app-versions : completed
time=2022-01-28 19:11:03 level=info msg=task 3: run : started
time=2022-01-28 19:11:03 level=error msg=unable to compile if clause stack-trace=below ...
::Trace:: expected bool, but got interface {}
Error: expected bool, but got interface {}
Usage:
  iter8 k run [flags]
```

It seems `if` statement is broken.

### LINKED ISSUE #1155: Using brew to install specific version of Iter8 CLI
Is it possible to install a specific version of Iter8 CLI? 

For other formulas installed using brew, there could be multiple versions, e.g.,

```
$ brew search postgresql@
==> Formulae
postgresql                   postgresql@11                postgresql@13                postgresql@9.5               qt-postgresql
postgresql@10                postgresql@12                postgresql@9.4               postgresql@9.6               postgrest
```

and then we can do `brew install postgresql@12`.

For iter8, this doesn't seem to be supported, i.e.,

```
$ brew search iter8@
==> Formulae
iter8-tools/iter8/iter8 ✔
```

### PR DESCRIPTION
**Pull request by sriumcp**
Fixes #1153 
Fixes #1155 

1. Bumping up test coverage to 70%
2. Minor improvements to release drafter
3. Minor improvements to contributing and readme markdowns
4. Experiment run test with if condition
5. Improvements to Iter8 hub command -- the latest stable release with matching major minor will be used for download
6. Removing private iter8 hub references
7. Enables brew install for a specific minor version
8. renaming load test to load test http (tutorial section)
9. Beating heart in homepage

### CODE REVIEW

**Comment by huang195** [comment]
@sriumcp thanks for the quick turn-around on this. This looks kind of massive, including a bunch of other things besides fixing the 2 issues I opened. I don't even know where to start to review this PR. I'd suggest that i)  we just merge the PR as is since v0.8 is still in the flux and we can afford some breaking changes, or ii) opening smaller PRs that are more suitable to do code reviews on.

File: basecli/hub.go [hunk 1]
```diff
@@ -74,10 +116,11 @@ iter8 hub -e tensorflow
...
-		// initialize the location of iter8hub
-		viper.BindEnv("ITER8HUB")
-		viper.SetDefault("ITER8HUB", Iter8Hub)
-		ifurl := path.Join(viper.GetString("ITER8HUB"), hubFolder)
+		hubRoot, err := getIter8Hub()
+		if err != nil {
+			return err
+		}
+		ifurl := path.Join(hubRoot, hubFolder)
 		log.Logger.Info("downloading ", ifurl)
```
Comment (kalantar) [inline basecli/hub.go:124]: The original allows use of a different hub by means of an environment variable. Haven't we lost this capability? It seems important.

File: mkdocs/docs/getting-started/install.md [hunk 2]
```diff
@@ -8,17 +8,45 @@ hide:
...
     ```shell
     brew tap iter8-tools/iter8
     brew install iter8
     ```
+    
+    ??? note "Install a specific version"
+        You can install the latest stable release of the Iter8 CLI with specific major and minor version numbers. For example, the following command installs the latest stable release of the Iter8 CLI with major `0` and minor `8`.
+        ```shell
+        brew tap iter8-tools/iter8
+        brew install iter8@0.8
```
Comment (kalantar) [inline mkdocs/docs/getting-started/install.md:22]: @v0.8 no?  The v is missing?

File: mkdocs/docs/getting-started/your-first-experiment.md [hunk 3]
```diff
@@ -2,28 +2,55 @@
 template: main.html
 ---
 
-# Load Test with SLOs
+# Your First Experiment
```
Comment (kalantar) [inline mkdocs/docs/getting-started/your-first-experiment.md]: We've flip flopped on the title before. What about "Your First Experiment: Load Test with SLOs"

File: mkdocs/docs/getting-started/your-first-experiment.md [hunk 4]
```diff
@@ -2,28 +2,55 @@
...
     brew tap iter8-tools/iter8
     brew install iter8
     ```
+    
+    ??? note "Install a specific version"
+        You can install the latest stable release of the Iter8 CLI with specific major and minor version numbers. For example, the following command installs the latest stable release of the Iter8 CLI with major `0` and minor `8`.
+        ```shell
+        brew tap iter8-tools/iter8
+        brew install iter8@0.8
+        ```
```
Comment (kalantar) [inline mkdocs/docs/getting-started/your-first-experiment.md]: Do we need this here? This is a tutorial. No need to provide every detail about install.

File: mkdocs/docs/getting-started/your-first-experiment.md [hunk 5]
```diff
@@ -119,69 +146,51 @@ View a report of the experiment in HTML or text formats as follows.
...
+
+        Latest observed values for metrics:
+        ***********************************
+
+          Metric                              |value
+          -------                             |-----
+          built-in/http-error-count           |0.00
+          built-in/http-error-rate            |0.00
+          built-in/http-latency-max (msec)    |186.56
+          built-in/http-latency-mean (msec)   |11.25
```
Comment (kalantar) [inline mkdocs/docs/getting-started/your-first-experiment.md:171]: I don't see changes to report.go that would modify the output in this way. Is that an oversight?

File: mkdocs/docs/getting-started/install.md (hunk 2 above)
Comment (sriumcp) [reply mkdocs/docs/getting-started/install.md:22]: No `v0.8`. Brew has a specific way to version software, and `major` and `minor` is that way. Adding a `v` in there will not work with Brew.

File: mkdocs/docs/getting-started/your-first-experiment.md (hunk 4 above)
Comment (sriumcp) [reply mkdocs/docs/getting-started/your-first-experiment.md]: Fixed.

File: basecli/hub.go (hunk 1 above)
Comment (sriumcp) [reply basecli/hub.go:124]: You mean this code which I added?

```go
// getIter8Hub gets the location of the Iter8Hub
 func getIter8Hub() (string, error) {
 	iter8HubTpl := "github.com/iter8-tools/iter8.git?ref=%v//hub"
 	viper.BindEnv("ITER8HUB")
 	iter8HubFromEnv := viper.GetString("ITER8HUB")
 	if len(iter8HubFromEnv) > 0 {
 		return iter8HubFromEnv, nil
 	}
 	tag, err := latestStableVersion()
 	if err != nil {
 		return "", err
 	}
 	return fmt.Sprintf(iter8HubTpl, tag), nil
 }
```

File: mkdocs/docs/getting-started/your-first-experiment.md (hunk 3 above)
Comment (sriumcp) [reply mkdocs/docs/getting-started/your-first-experiment.md]: Yes, I like "Your First Experiment: Load Test an HTTP Service with SLOs"

File: mkdocs/docs/getting-started/your-first-experiment.md (hunk 5 above)
Comment (sriumcp) [reply mkdocs/docs/getting-started/your-first-experiment.md:171]: The correct template was already merged in the previous PR. This is just an update to the documentation.

File: mkdocs/docs/getting-started/your-first-experiment.md (hunk 3 above)
Comment (sriumcp) [reply mkdocs/docs/getting-started/your-first-experiment.md]: Sorry about further flip-flopping. 

I accepted your suggestion in this PR and already pushed the change. But when I view it, it is making the title overflow onto the next line.

Anyway, I will fix this issue in  a subsequent PR. Flip-flopping on the title is less of an issue (we don't want to flip-flop on the name of the file as that starts breaking links badly).

**Review by kalantar — APPROVED** [review]
lgtm

**Review by kalantar — APPROVED** [review]
lgtm

**Pull request merged**